src/content/docs/kubernetes-pixie/kubernetes-integration/get-started/kubernetes-integration-compatibility-requirements.mdx

```diff
@@ -9,41 +9,42 @@ redirects:
   - /docs/integrations/kubernetes-integration/get-started/kubernetes-integration-compatibility-requirements
 ---
 
-[New Relic's Kubernetes integration](/docs/integrations/kubernetes-integration/get-started/introduction-kubernetes-integration) can be [installed directly](/docs/integrations/kubernetes-integration/installation/kubernetes-installation-configuration#install) on a server or VM, or through several [cloud platforms](/docs/integrations/kubernetes-integration/installation/kubernetes-installation-configuration#cloud-platforms), such as GKE, EKS, AKS, or OpenShift. Each has a different compatibility with our integration.
+The [Kubernetes integration](/docs/integrations/kubernetes-integration/get-started/introduction-kubernetes-integration) is compatible with many different platforms including GKE, EKS, AKS, OpenShift, and more. Each has a different compatibility with our integration. You can find more information in this page.
+
+## Requirements [#reqs]
+
+The New Relic Kubernetes integration has the following requirements:
+
+* A New Relic account. Don't have one? [Sign up for free](https://newrelic.com/signup). No credit card required.
+* Linux distribution [compatible with New Relic infrastructure agent](/docs/infrastructure/new-relic-infrastructure/getting-started/compatibility-requirements-new-relic-infrastructure#operating-systems).
+* [`kube-state-metrics`](https://github.com/prometheus-community/helm-charts/tree/main/charts/kube-state-metrics) running on the cluster.
```

Comment:
I would get rid of this entirely. We automatically install `kube-state-metrics` as a dependency. User installing a separate instance can, in fact, lead to problems.

While it is possible for users to disable the ksm we include, and to bring their own, that is a pretty advanced use case that is only borderline supported. I don't think that should be on the front page.

Reply:
I've removed it because @csongnr agrees with your comment.